.circleci/config.yml

```diff
@@ -1,39 +1,51 @@
-version: 2
-jobs:
-  build:
+---
+version: 2.1
+executors:
+  executor_med:  # 2cpu, 4G ram
     docker:
-      - image: circleci/openjdk:11.0.2-jdk-stretch
+      - image: circleci/openjdk:11.0.4-jdk-stretch
       - image: postgres:11.4-alpine
         environment:
           POSTGRES_USER: test
           POSTGRES_PASSWORD: test
           POSTGRES_DB: orion
-
-    working_directory: ~/repo
-
+    resource_class: medium
+    working_directory: ~/project
     environment:
-      TERM: dumb
       JAVA_TOOL_OPTIONS: -Xmx2048m
-      GRADLE_OPTS: -Dorg.gradle.daemon=false -Dorg.gradle.workers.max=2
-      GRADLE_MAX_TEST_FORKS: 2
+      GRADLE_OPTS: -Dorg.gradle.daemon=false -Dorg.gradle.parallel=true -Dorg.gradle.workers.max=2 -Xmx2048m
       POSTGRES_USER: test
       POSTGRES_PASSWORD: test
       POSTGRES_DB: orion
 
+  executor_machine:
+    machine:
+      image: ubuntu-1604:201903-01 #Ubuntu 16.04, docker 18.09.3, docker-compose 1.23.1
+      docker_layer_caching: true
+    working_directory: ~/project
+
+commands:
+  prepare:
+    description: "Prepare"
     steps:
       - checkout
-
       - run:
-          name: Install Sodium Dependencies
+          name: Install Packages - LibSodium
```

Comment:
We should rename this to `Install Dependencies`. We aren't only installing LibSodium, are we?

Reply:
done